libs/auth-api-lib/src/lib/entities/dto/base/client-base-dto.ts

```diff
@@ -8,6 +8,20 @@ import {
 import { ApiProperty } from '@nestjs/swagger'
 
 export abstract class ClientBaseDTO {
+  @IsString()
```

Comment:
Should this file be named `client-base.dto.ts` to follow the same conventions as others filenames? (e.g. `client.model.ts`)

Reply:
Will fix in another commit